### ISSUE TITLE
Cookies cleared when application closed

### ISSUE THREAD

**Issue by pemmoura** [flathub/org.chromium.Chromium#45]
Using version 87.0.4280.88, every time I close and reopen chromium all sessions are lost, have to login into every account.

**Comment by pemmoura** [comment]
Not sure what changed but is working now.

**Comment by AvantR** [comment]
I have this issue too. For me isn't working

**Comment by Latesil** [comment]
same problem here

**Comment by refi64** [comment]
1. Do any of you observe any interesting console output when this happens?
2. Is anyone signed into Google? If so, do you remain signed in when this
occurs?

**Comment by Latesil** [comment]
1.  I don't know if it interesting or related to the problem but I have a lot of that: 

> Failed to load resource: net::ERR_BLOCKED_BY_CLIENT

2. I wasn't signed on Google. But anyway when I reopen browser I am unsigned from Google.

**Comment by refi64** [comment]
Any other console output? That logging not particularly unusual iirc

**Comment by rany2** [comment]
@refi64 If it helps I had this same issue before on XFCE. I solved this by uninstalling gnome-keyring. It's not exactly a solution but it might help you figure out the root of the problem

**Comment by cjao** [comment]
I encounter this on Fedora 33. Chromium outputs a ton of errors like
```
[44:59:0127/210030.055038:ERROR:database.cc(1707)] Cookie sqlite error 5, errno 0: database is locked, sql: DELETE FROM cookies WHERE host_key = ?
[44:59:0127/210030.071244:ERROR:database.cc(1707)] Cookie sqlite error 5, errno 0: database is locked, sql: DELETE FROM cookies WHERE host_key = ?
[44:59:0127/210030.087973:ERROR:database.cc(1707)] Cookie sqlite error 5, errno 0: database is locked, sql: DELETE FROM cookies WHERE host_key = ?
[44:59:0127/210030.103650:ERROR:database.cc(1707)] Cookie sqlite error 5, errno 0: database is locked, sql: DELETE FROM cookies WHERE host_key = ?
```

**Comment by dsd** [comment]
I saw this when installing Chromium on a fresh copy of Endless OS around 8 days ago - got signed out of google every time I closed the browser.
However coming back a week later, I updated to the latest flatpak version and the problem went away.
Notably when the problem went away, I was able to access my google account without logging in. So, the cookie data had been stored OK in the earlier version 8 days ago.
